src/frontend/src/features/layouts/components/thread-view/components/attachment-preview-modal/_index.scss

```
@@ -0,0 +1,163 @@
+    border: 1px solid var(--c--contextuals--border--surface--primary);
+}
+
+.attachment-preview-sidebar__file-meta {
+    min-width: 0;
+    flex: 1;
+}
+
+.attachment-preview-sidebar__file-name {
+    margin: 0;
+    font-weight: 600;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)